Hazel/src/Hazel/Renderer/RendererAPI.cpp

```diff
@@ -1,8 +1,22 @@
 #include "hzpch.h"
-#include "RendererAPI.h"
+#include "Hazel/Renderer/RendererAPI.h"
+
+#include "Platform/OpenGL/OpenGLRendererAPI.h"
 
 namespace Hazel {
 
 	RendererAPI::API RendererAPI::s_API = RendererAPI::API::OpenGL;
 
+	Scope<RendererAPI> RendererAPI::Create()
+	{
+		switch (s_API)
+		{
+			case RendererAPI::API::None:    HZ_CORE_ASSERT(false, "RendererAPI::None is currently not supported!"); return nullptr;
+			case RendererAPI::API::OpenGL:  return CreateScope<OpenGLRendererAPI>();
+		}
+
+		HZ_CORE_ASSERT(false, "Unknown RendererAPI!");
+		return nullptr;
+	}
```

Comment:
We should maybe consider having this create method take in the API as a parameter and then set `RendererAPI::s_API` along with the other code. It might, however, also be better to do this somewhere else and I think it falls outside the scope of this PR.

Reply:
The way I was thinking about it was have 1 function that actualy sets `RendererAPI::s_API`. This would usualy be read in from some config settings that are saved on the user side, if not, use a default (probably OpenGL).

I bet cherno will have a solution by that time. Whatever happens, it is not part of this PR as you said. Marking it as resolved.